File: we_4_9_why_temperature_calibration_matters.m
% Integrated Navigation Systems — Chapter 4: Inertial Sensors and Error Modeling
% Worked Example 4.9 — Why temperature calibration matters
% Companion code (educational reference implementation; see the book for context).

kT=0.005; dT=20; db=kT*dT;
fprintf('bias shift=%.2f deg/s = %.0f deg/hr\n',db,db*3600);  % 0.10, 360

File: we_4_9_why_temperature_calibration_matters.py
# Integrated Navigation Systems — Chapter 4: Inertial Sensors and Error Modeling
# Worked Example 4.9 — Why temperature calibration matters
# Companion code (educational reference implementation; see the book for context).

kT, dT = 0.005, 20; db=kT*dT
print(f'bias shift={db:.2f} deg/s = {db*3600:.0f} deg/hr')  # 0.10 360
